packages/visx-tooltip/src/hooks/useTooltipInPortal.tsx

```diff
@@ -36,6 +36,11 @@ export default function useTooltipInPortal({
   ...useMeasureOptions
 }: UseTooltipPortalOptions | undefined = {}): UseTooltipInPortal {
   const [containerRef, containerBounds, forceRefreshBounds] = useMeasure(useMeasureOptions);
+  const [isSsr, setIsSsr] = useState(false)
```

Comment:
@korompaiistvan looks like a couple minor linting errors (just missing `;`)

```suggestion
  const [isSsr, setIsSsr] = useState(false);
```